OpenRA.Game/FileFormats/PngLoader.cs

```diff
@@ -111,6 +111,9 @@ public static Bitmap Load(Stream s)
 
 								case "IEND":
 									{
+										if (bitmap == null)
+											throw new InvalidDataException("Image header not found.");
```

Comment:
Looks like it's not clever enough. I'll drop this and mark it as a false positive.